unpackaged/functional_tests/address/classes/Address_FTST.cls

```diff
@@ -290,4 +310,114 @@ public with sharing class Address_FTST {
             'The current address is incorrect'
         );
     }
+
+    /**
+     * @description T-3904085: Using Household account model, modify an existing Contact Address and assess new address
```

Comment:
Just a reminder to check the 'automate' checkbox for test scenario. Please ignore if you already did.

Reply:
Thanks for reminding me, I just changed the execution type to automated.